schemas/donor.json

```diff
@@ -280,6 +280,17 @@
         "foreignKey": "primary_diagnosis.submitter_primary_diagnosis_id",
         "validationDependency": true
       }
+    },
+    {
+      "description": "Indicate the donors' country of residence at the time they were enrolled in the study.",
```

Comment:
`Indicate the donor's country of residence at the time they were enrolled in the study.`

In all other fields we refer to donor singular instead of plural

Reply:
Should we include the source of country values in description?